src/dialect/mod.rs

```diff
@@ -691,64 +639,40 @@ mod tests {
                 self.0.identifier_quote_style(identifier)
             }
 
-            fn supports_string_literal_backslash_escape(&self) -> bool {
-                self.0.supports_string_literal_backslash_escape()
-            }
-
             fn is_proper_identifier_inside_quotes(
                 &self,
-                chars: std::iter::Peekable<std::str::Chars<'_>>,
+                chars: Peekable<Chars<'_>>,
```

Comment:
it would be great if clippy or rustfmt could warn about these unnecessary paths, rustrover warns, hence the drive by change.